tesseract_core/sdk/tesseract.py

```diff
@@ -455,3 +527,59 @@ def _run_tesseract(self, endpoint: str, payload: dict | None = None) -> dict:
             endpoint = "openapi.json"
 
         return self._request(endpoint, method, payload)
+
+
+class LocalClient:
```

Comment:
Shall we leave this as an undocumented feature? It's very useful, but could be confusing in the public api

Reply:
"this" == `from_tesseract_api`? Or `LocalClient`?

The latter is purely internal so I wouldn't really advertise that for anything. The former, I'm not sure... we could leave it undocumented for now and see how we like using it?

Reply:
I was referring to LocalClient; ok! do consider whether we want to rename it as `_LocalClient`, but I am fine with both options

Reply:
(feel free to resolve this thread and press merge after you make a call on this)

Reply:
Actually, I'll do it :)

Reply:
I see. We don't use `_` names consistently in this codebase, particularly for classes, so either is fine with me.